MatrixSDK/Utils/MXBugReportRestClient.m

```diff
@@ -201,25 +201,6 @@ -(void)sendBugReport:(NSString *)text sendFiles:(NSArray<NSURL*>*)files attachGi
             [formData appendPartWithFormData:[self.build dataUsingEncoding:NSUTF8StringEncoding] name:@"build"];
         }
 
-#if __has_include(<MatrixKit/MatrixKit.h>)
```

Comment:
All of this has been moved over to the [app-side](https://github.com/vector-im/element-ios/pull/7458/files#diff-cde9f6c3a97bd7fe4f0ffb9b0e096e8c582d6484ae2dfe057fefd3d28bf4a4e6R71-R77)